packages/okta-react-native/README.md

```diff
@@ -32,12 +25,9 @@ In Okta, applications are OpenID Connect clients that can use Okta Authorization
 
 | Setting             | Value                                        |
 | ------------------- | -------------------------------------------- |
-| App Name            | My Native App                                |
-| Login redirect URIs | com.oktapreview.{yourOrg}:/+expo-auth-session|
-|                     | exp://localhost:19000/+expo-auth-session     |
-| Grant Types Allowed | Authorization Code, Refresh Token            |
-
-**Note:** *Login redirect URIs will depend on your scheme. For setting up a scheme on Android and iOS, refer to [configuration](#configuration).*
+| App Name            | {YOUR APP NAME}                              |
+| Login redirect URIs | {YOUR_REDIRECT_URL}                          |
+| Grant Types Allowed | {PICK_YOUR_SCOPES}                           |
```

Comment:
Are these terms defined anywhere?  Do we have a doc page we can point a user to where they can figure out what these values would be? 

I'm weak on mobile, but would https://developer.okta.com/docs/guides/sign-into-mobile-app/ be a helpful thing, or does that not cover these values?

Reply:
These aren't defined, the closest I found is https://developer.okta.com/docs/guides/sign-into-mobile-app/android/create-okta-application/, but this is only for android setup. maybe we should introduce a setup for react native only?

Reply:
Probably, I'd say create a JIRA ticket (leave it unassigned) and Rob/Nate/Albert will prioritize it.  (It is probably chicken/egg waiting on these changes)

Meanwhile, for this PR, you should give at least a little guidance on how to figure out what these values are - either a place in the docs to look, a place in the the okta dashboard to look, or a sample value.  That reduces developer frustration when trying to follow your README when they aren't super-familiar with using Okta.

Reply:
I added some sample values, let me know if it looks good